src/pages/StudyDetail/index.tsx

```diff
@@ -0,0 +1,155 @@
+import { useEffect, useState } from 'react';
+import { StudyDetailContainer } from '@pages/StudyDetail/style';
+import { Button } from '@mui/material';
+import { studyDetailQusetionType } from '@interfaces/studyDetailQuestion';
+import { detailMemberType, studyDetailType } from '@interfaces/studyDetail';
+import {
+  StudyDetailMbtiRecommand,
```

Comment:
```suggestion
  StudyDetailMbtiRecommend,
```
오탈자가 있습니다! `Recommand`로 적힌 부분이 여럿 있는거 같은데 이쪽에만 코멘트 남겨두겠습니다.